docs-content/getting-started/fullstack-frameworks/next-js/2_page-router.md

```diff
@@ -139,11 +139,52 @@ function ThrowerOfErrors({
 }
 ```
 
-## Catch server-side render (SSR) errors (optional)
+## Enable server-side tracing
```

Comment:
It turns out that `instrumentation.ts` captures traces, but only in certain cases can it capture SSR errors. So `_error.tsx` and `error.tsx` are still the most reliable ways to capture SSR errors.

In short, `instrumentation.ts` doesn't currently catch Edge SSR. Our `registerHighlight` handler is currently empty in Edge environments, but OpenTelemetry doesn't really work in Edge anyway, so I don't suspect that Next.js instruments Edge routes anyway.